src/pynws/simple_nws.py

```diff
@@ -263,19 +265,29 @@ async def update_observation(
 
     async def update_forecast(self: SimpleNWS, *, raise_no_data: bool = False) -> None:
         """Update forecast."""
-        forecast = await self.get_gridpoints_forecast()
+        forecast_with_metadata = await self.get_gridpoints_forecast()
+        forecast = forecast_with_metadata["periods"]
         if self._filter_forecast(forecast):
             self._forecast = forecast
+            self._forecast_metadata = {
+                metadataKey: forecast_with_metadata[metadataKey]
```

Comment:
I've approved this already, but my final question is whether we think there will always be these keys in the meta data?  I'm wondering if we should use `.get` in case it is randomly missing?

Reply:
```suggestion
                metadataKey: forecast_with_metadata.get(metadataKey)
```

Reply:
Ah yea, totally agree.